app/policies/group_policy.rb

```diff
@@ -83,6 +83,15 @@ def change_leader?
     edit?
   end
 
+  def select_suite?
```

Comment:
So this policy will change in #276? Or should we restrict selection to groups that are the next groups in their respective draws?